src/user/services/user.service.spec.ts

```diff
@@ -35,7 +34,7 @@ describe('UserService', () => {
       providers: [
         UserService,
         {
-          provide: getRepositoryToken(User),
+          provide: UserRepository,
```

Comment:
@war1oc 
Any reason for doing this?
Now our service unit tests seem to have become aware that UserRepository class exists? 
Let me know your thoughts.

Reply:
Tests were failing for this service because the dependency was invalid. We are using UserRepository dependency in the service but passing something else.

> Now our service unit tests seem to have become aware that UserRepository class exists?
I'm not sure I got this part, but the tests need to resolve the correct dependency and that goes into provide, the value is still a mock.

This is how it's done in article module too btw.